scripts/gke/deploy.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# This script creates a kubeflow deployment on GCP
+# It checks for kubectl, gcloud, ks
+# Uses default PROJECT, ZONE, EMAIL from gcloud config
+# Creates a deployment manager config copy and edits appropriate values
+# Adds user to the IAP role
+# Creates the deployment
+# Creates the ksonnet app, installs packages, components and then applies them
+
+set -xe
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPT_DIR}/../util.sh"
+KUBEFLOW_REPO=$(cd "${SCRIPT_DIR}/../.."; pwd)
+check_install gcloud
+check_install kubectl
+check_install ks
```

Comment:
Might want to check ks version since only new one support local registry.

Reply:
This is non-trivial to do in bash. Added a TODO. We will mention in the docs that the requirements are `ks > 0.11`